src/PayPal/Api/Subscription/Link.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Subscription;
+
+class Link extends \Swag\PayPal\PayPal\Api\Common\Link
```

Comment:
we normally import the Link with the alias `CommonLink`

Reply:
done